backend/__tests__/routes/doctors.test.js

```diff
@@ -76,7 +76,7 @@ describe("PATCH: update the priority of a patient as a doctor", () => {
     it("Returns 200: Priority is updated successfully by a doctor", async () => {
         const data = {
             accountId: 239,
-            patientId: 3,
+            patientId: 7,
```

Comment:
should we move this 7 to a constant?

Reply:
The repetive use of a non-constant was there before, but I think it might be better to change it in a proper refactoring task later.